timelapse/encoder.py

```diff
@@ -56,6 +56,7 @@ def run(self) -> None:
                     command, capture_output=True, check=True)
             except subprocess.CalledProcessError as e:
                 notify("Timelapse Error:", e.stderr.decode('utf-8'))
+                notify("Timelapse: ffmpeg not found.", e.stderr.decode('utf-8'))
```

Comment:
@ItsThompson Thanks for updating the PR.

I suppose we now have two consecutive `notify`s. We can probably delete the old one (i.e. the first one)?

Reply:
Because there are many different errors, wont ln 58 be necessary to know the error number to understand what when wrong?

Reply:
Yeah, we should remove the first `notify`. 

Otherwise this would just show two notifications which differ only in the headline. One would be "Timelapse Error:", the other would be "Timelapse: ffmpeg not found.". But in both cases, the error would be the same and we print that already (that's the `e.stderr.decode('utf-8')` part). Hope that makes sense.

Could you remove the first `notify` @ItsThompson?

Reply:
Will do